lib/unhangout-sockets.js

```diff
@@ -512,12 +613,25 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
             });
         });
         this.db.events.on("change:sessionsOpen", function(event, isOpen) {
+            console.log("SESSION  : " + isOpen);
```

Comment:
Please remove console.log statements from final code.  If you need to log, use "logger.debug" and so forth.  That way, we can automatically suppress debug statements in production and testing environments.